setup.py

```diff
@@ -21,7 +21,13 @@
 
 setup_requirements = ['pytest-runner', ]
 
-test_requirements = ['pytest', ]
+tests_require = [
```

Comment:
Since `test_requirements` has been renamed as `tests_require`, please also rename `requirments` as `install_requires` and `setup_requirements` as `setup_requires`.

Reply:
True. Changes already commited.